commands-yml/commands/mobile-command.yml

```diff
@@ -0,0 +1,129 @@
+---
+  name: Execute Mobile Command
+  short_description: Execute a native mobile command
+  
+  description:
+    |
+      Execute a variety of native, mobile commands that aren't associated with a specific endpoint
+
+      Syntax is `execute("mobile: <commandName>", <JSON serializable argument>)` 
+
+      List of available commands:
+
+      ### IOS
+      | Command | Description | Argument | Argument Example |
+      | ------- | ----------- | -------- | ---------------- |
+      | mobile:viewportScreenshot | Like [screenshot](/commands/session/screenshot/) but only includes contents of viewport | <none> | <none> |
+
+      ### IOS (XCUITest Only)
+      * [IOS XCTest Mobile Gestures](/writing-running-appium/ios/ios-xctest-mobile-gestures)
```

Comment:
still think we should list the `mobile: xxx` name here for reference, and _theN_ link to it